crates/bevy_wgpu/Cargo.toml

```diff
@@ -29,7 +29,7 @@ bevy_winit = { path = "../bevy_winit", optional = true, version = "0.5.0" }
 bevy_utils = { path = "../bevy_utils", version = "0.5.0" }
 
 # other
-wgpu = "0.9"
+wgpu = { version = "0.9", features = ["spirv"] }
```

Comment:
Could you make that a git dependency to wgpu? While some jobs of CI will fail because of that, the others should work a little better

Reply:
Updated. For the glTF example to run we also need to patch Naga so I added a patch in there too. Since we can't actually merge until WGPU publishes another release, we can just take that and the git dependencies out when WGPU 0.10 comes out.